district_data_import.py

```diff
@@ -144,6 +146,53 @@
             'ZIP': district_record['zip']
         })
 
+        # Update for potential switch of parent name order
+
+        curr_informal_sal = dp_donorrecord['INFORMAL_SAL']
+        curr_main_f_name = dp_donorrecord['FIRST_NAME']
+        curr_spouse_f_name = dp_donorrecord['SP_FNAME']
+
+        dp_donorrecord_with_pnames = district_data_utils.create_dp_donorrecord(district_record=district_record, school_year=args.school_year)
+        dp_donorrecord.update({
+            'FIRST_NAME': dp_donorrecord_with_pnames['FIRST_NAME'],
+            'LAST_NAME': dp_donorrecord_with_pnames['LAST_NAME'],
+            'SP_FNAME': dp_donorrecord_with_pnames['SP_FNAME'],
+            'SP_LNAME': dp_donorrecord_with_pnames['SP_LNAME'],
+            'EMAIL': dp_donorrecord_with_pnames['EMAIL'],
+            'SPOUSE_EMAIL': dp_donorrecord_with_pnames['SPOUSE_EMAIL'],
+            'MOBILE_PHONE': dp_donorrecord_with_pnames['MOBILE_PHONE'],
+            'SPOUSE_MOBILE': dp_donorrecord_with_pnames['SPOUSE_MOBILE'],
+            'SALUTATION': dp_donorrecord_with_pnames['SALUTATION'],
```

Comment:
You are blindly updating `SALUTATION` but conditionally updating `INFORMAL_SAL`. The latest comments from Byung/Felicia indicate these fields should be consistent with one another.

Reply:
It is fine to update the salutation blindly, as it *should* blindly follow the parent order--
 there have not been any personalizations/customizations here. However, for informal_sal, we have to use logic to see if the informal_sal in DP was auto-generated. If auto-generated, then we can blindly follow parent orded. If not auto-generated, we want to leave it alone.

Reply:
In one of the many emails we have, Byung said 'I do think the salutation and informal_sal fields should be consistent with each other.' If you blindly update `SALUTATION` and conditionally update `INFORMAL_SAL`, this will not be the case.